### PR TITLE
Fixed broken links, and updated some to https

### PR DESCRIPTION
**Pull request by tophj-ibm**
This PR fixes a couple of links and typos I noticed in the docs merge pr (sorry, was merged before I could comment). 

ping @thaJeztah @moxiegirl @SvenDowideit 

Signed-off-by: Christopher Jones tophj@linux.vnet.ibm.com

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ own.
 6.  Check your writing for style and mechanical errors.
 
 	Use our [documentation style
-	guide](/opensource/doc-style.md) to check style. There are
+	guide](https://docs.docker.com/opensource/doc-style/) to check style. There are
```
Comment (thaJeztah) [inline docs/README.md]: A,h, well spotted; the README.md will not be rendered by Hugo

**Comment by thaJeztah** [comment]
Thanks! LGTM

ping @moxiegirl for the merge

**Comment by MHBauer** [comment]
Yay https!

**Comment by moxiegirl** [comment]
LGTM thank you @tophj-ibm for the contribution,

**Pull request merged**